src/api/folderResourceMetaApi.ts

```diff
@@ -28,8 +29,18 @@ import {
   GQLQueryFolderResourceMetaSearchArgs,
 } from "../types/schema";
 import { articleToMeta, learningpathToMeta } from "../utils/apiHelpers";
+import { getArticleIdFromUrn } from "../utils/articleHelpers";
 
-type MetaType = "article" | "learningpath" | "multidisciplinary" | "concept" | "image" | "audio" | "video" | "folder";
+type MetaType =
+  | "article"
+  | "learningpath"
+  | "multidisciplinary"
+  | "concept"
+  | "image"
+  | "audio"
+  | "video"
+  | "folder"
```

Comment:
Folder kan fjernes fordi vi lagrer det på en anna måte.

Reply:
Kan denne egentlig byttes ut med ResourceTypes fra myndla-api-typescript?

Reply:
Ja. Vi har eksplisitt støtte for henting av folders i samme fil

Reply:
Men folder-metadata blei vel aldri innført som type å favorittmerke.

Reply:
Ser ut som at det har å gjøre med favorittmerking av mapper https://github.com/NDLANO/graphql-api/pull/448

Reply:
Eg er klar over det men vi favorittmerker ikkje mapper. Vi lagrer dei. Dette blei aldri tatt i bruk.

Reply:
Da kan det fjernes. Foreslår at vi tar det i en ny PR isåfall!

Reply:
Uenig. Bedre å slette det og bumpe backend-typer slik at dette blir korrekt.

Reply:
Enig i at det bør gjøres, men det har ikke noe med denne PR'en å gjøre. Ryddig å separere PR'ene våre til å kun handle om en ting om gangen.